### PR TITLE
Fix master

### PR DESCRIPTION
**Pull request by jfragoulis**
`bundle exec rake` is producing some errors. These commits address these errors.

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -12,8 +12,7 @@ AllCops:
     - 'tmp/**/*'
     - 'spec/smoke_tests/**/*.rb'
 
-# See https://github.com/rubocop-hq/rubocop/issues/6410
```
Comment (pirj) [inline .rubocop.yml]: Do you mind to keep this reference? I believe it's here to explain why this is disabled.

File: lib/rubocop/cop/rspec/described_class.rb [hunk 2]
```diff
@@ -139,7 +139,7 @@ def offensive?(node)
           if style == :described_class
             offensive_described_class?(node)
           else
-            node.send_type? && node.method_name == :described_class
+            node.send_type? && node.method?(:described_class)
```
Comment (pirj) [inline lib/rubocop/cop/rspec/described_class.rb]: Will this work with the earliest RuboCop version that we support, 0.68.1?

Comment (pirj) [reply lib/rubocop/cop/rspec/described_class.rb]: With 0.68.1 installed `./spec/rubocop/cop/rspec/implicit_subject_spec.rb` fails several examples with:
```
      NoMethodError:
        undefined method `method?' for #<RuboCop::AST::BlockNode:0x00007ff05a4aaef0>
        Did you mean?  method
                       methods
```

Do you mind to keep those intact and disabling the cop that suggested this to `.rubocop_todo.yml`?

Comment (jfragoulis) [reply lib/rubocop/cop/rspec/described_class.rb]: Hmm, what more can if `bundle exec rubocop` passes?

**Comment by bquorning** [comment]
Ah, issues between Rubocop v0.76.0 and v0.77.0. Thank you for fixing.

**Pull request merged**